### ISSUE TITLE
Add total number of played games per player

### ISSUE THREAD

**Issue by jonium** [eduherminio/LichessTournamentAggregator#25]
The result could also give the total played game (per player).
Sometimes, in fact, players join the tournaments after it has begun, or left it before the end.

**Comment by eduherminio** [comment]
I agree with your suggestion and I'd definitely implement straight away it if Lichess results API provided that extra info, _but_;

Lichess results API (currently) provides the following information:
* Arena tournaments
  `{"rank":1,"score":20,"rating":2249,"username":"Kimonlio","performance":2447}` 
* Swiss tournaments
  `{"rank":1,"points":6,"tieBreak":22.5,"rating":2304,"username":"bulletbaz","performance":2444}`

Therefore, I'm afraid it's not trivial to know the number games played per player.

I can think of a way to get it: downloading all the tournaments games, parsing them (pgn) and matching them to the players.
However, that would:
* Impact runtime ([click here](https://lichess.org/tournament/LRxqoah3) to test how long it takes to download a bunch of games using the API).
* Take good amount of developer effort.

-----
In short,
* **I'm afraid I don't find this feature feasible to be implemented right now**, with the existing API.
* I've opened https://github.com/lichess-org/api/issues/110 to request that information to be included in the Lichess results API.
  Feel free to 👍🏼 it if and/or express there your thoughts about having the number of played games per player available.
  If that feature request gets implemented, it'll be easy to incorporate that info to `LichessTournamentAggregator`.

**Comment by eduherminio** [comment]
That info [is not going to be added to the API any time soon](https://github.com/lichess-org/api/issues/110) due to the same reasoning behind not implementing it outside of it:

> This can only done by adding a lot of computation on lichess side, and for dubious reasons? I don't think most API users want it.
> 
> If it were cheap I would add it, but it's not.

So I'm afraid this one is a `Won't do` for me as well.
I'll close the issue for now, but will take the idea into consideration if an 'advanced report' is ever created.

I'd suggest you to compare `Total tie breaks` between the players with the same points in a swiss tournament to get an idea of which player could have played less games, but you're probably already doing it.

**Comment by jonium** [comment]
ok, thanks
